### DISCUSSION TITLE
Getting Error, While Debug the  GAUGE specs in  PYTHON  Via VS- CODE

### DISCUSSION THREAD

**Discussion by LogeshS-alation** [getgauge/gauge#1993] [General]
[Gauge]
Unable to SetReadDeadLine on runner: set tcp 127.0.0.1:52652: use of closed network connection

Get Support ----------------------------
	Docs:          https://docs.gauge.org
	Bugs:          https://github.com/getgauge/gauge/issues
	Chat:          https://github.com/getgauge/gauge/discussions

Your Environment Information -----------
	darwin, 1.1.8
	html-report (4.0.12), java (0.7.15), python (0.3.4), screenshot (0.0.1), xml-report (0.2.3)
Error: Tests failed.

**Comment by sriv** [comment]
Can you share some more info? When do you see this error? What language does your project use? Some steps to reproduce this issue would be good.

**Reply by LogeshS-alation** [reply]
@sriv ,

Currently, we are using the python language with the selenium tool with gauge frame wok in the VS-Code IDE.

Step 1: Installed  Gauge Extension  in vs-code
Step 2:  Make  Some Breakpoint  in the step_implementation file
E.g.:
![image](https://user-images.githubusercontent.com/83263892/118479249-6b35d200-b72e-11eb-9a20-86fefd7a3dfe.png)

Step 3: Click Debug scenario option and verify the debug prompt in the Vs-Code. IDE

![image](https://user-images.githubusercontent.com/83263892/118478263-4bea7500-b72d-11eb-9c90-bafeb7c807ce.png)

Expected Result: Geeting error in the console.

![image](https://user-images.githubusercontent.com/83263892/118479808-18104f00-b72f-11eb-87cf-2769ceb5827c.png)

**Reply by zabil** [reply]
Please try upgrading to the latest version of Gauge python which is 0.3.15

**Reply by LogeshS-alation** [reply]
@zabil 

Thanks for the Note.
Followed by  your inputs   i upgraded  the GaugePython version. it's currently working now.
